setup.py

```diff
@@ -1,21 +1,15 @@
 #!/usr/bin/env python
-
 from setuptools import setup, find_packages
-from distutils.core import setup
 
-setup(name='python_etl',
-      version='0.1',
-      description='Python ETL jobs for Firefox Telemetry to be scheduled on Airflow.',
-      author='Ryan Harter',
-      author_email='harterrt@mozilla.com',
-      url='https://github.com/mozilla/python_etl.git',
-      packages=find_packages(exclude=['tests']),
-      install_requires=[
-          'python_moztelemetry',
-          ],
-      test_requires=[
```

Comment:
Hm, they shouldn't, test_requires are only used when you run the tests via `python setup.py test`, which we don't. In python_moztelemetry there is a magic pytest running used for setuptools (see the `pytest-runner` entry in `setup_requires`). Which will then pick up the `test_requires` parameter.